tests/test_ir.py

```diff
@@ -161,8 +161,8 @@ def test_op_operands_assign():
     op.operands = [val2, val1]
     op.verify()
 
-    assert len(val1.uses) == 1
-    assert len(val2.uses) == 1
+    assert len(list(val1.uses)) == 1
```

Comment:
Why are we preferring `len(list(*.uses))` to `*.get_length()`

Reply:
Also this one should be `has_single_use` anyway

Reply:
Yes, fixed!